src/components/content/policies/Policies.tsx

```diff
@@ -0,0 +1,297 @@
+    };
+    const showPoliciesManagementServiceDetails = (record: PolicyVo) => {
+        setId(record.id);
+        setIsOpenPolicyDetailsModal(true);
+    };
+    const closePoliciesManagementServiceDetailsModal = () => {
+        setIsOpenPolicyDetailsModal(false);
+    };
+    const updatePoliciesManagementService = (record: PolicyVo) => {
+        setCurrentPolicyService(record);
+        setIsOpenEditPolicyModal(true);
+    };
+
+    return (
+        <>
+            <div className={'services-content policies-instance-list'}>
+                {deletePoliciesManagementServiceRequest.isSuccess && id.length > 0 ? (
+                    <PolicyDeleteResultStatus
+                        id={id}
+                        isError={deletePoliciesManagementServiceRequest.isError}
+                        isSuccess={deletePoliciesManagementServiceRequest.isSuccess}
+                        error={deletePoliciesManagementServiceRequest.error}
+                        setIsDeletingCompleted={setIsDeletingCompleted}
+                        getDeleteCloseStatus={getDeleteCloseStatus}
+                    />
+                ) : null}
+                <div className={'add-policy-container'}>
+                    <div className={'update-policy'}>
+                        <Button
+                            type='primary'
+                            onClick={refreshPoliciesManagementServiceList}
+                            icon={<SyncOutlined />}
+                            disabled={
+                                (deletePoliciesManagementServiceRequest.isPending ||
+                                    deletePoliciesManagementServiceRequest.isSuccess ||
+                                    deletePoliciesManagementServiceRequest.isError) &&
+                                !isDeletingCompleted
+                            }
+                        >
+                            refresh
+                        </Button>
+                    </div>
+
+                    <div className={'add-policy'}>
+                        <Button
+                            onClick={addPoliciesManagementService}
+                            type='primary'
+                            icon={<AppstoreAddOutlined />}
+                            disabled={
+                                (deletePoliciesManagementServiceRequest.isPending ||
+                                    deletePoliciesManagementServiceRequest.isSuccess ||
+                                    deletePoliciesManagementServiceRequest.isError) &&
+                                !isDeletingCompleted
+                            }
+                        >
+                            add
+                        </Button>
+                    </div>
+                </div>
+
+                <Modal
+                    title={'Edit Policy'}
+                    width={1000}
+                    footer={null}
+                    open={isOpenEditPolicyModal}
```

Comment:
in case of update, we must fill all the existind data, including the policy content. 
If the user only wants to update the 'enabled' status, then we send back same policy content in the update request. User need not upload the file again.